src/collar/oc_settings.lsl

```diff
@@ -15,6 +15,90 @@ https://github.com/OpenCollarTeam/OpenCollar
 
 //string g_sParentMenu = "Apps";
 
+
```

Comment:
I see this MR is targeted against master.  You might want to target it against the q3 feature branch to reduce the likelihood of merge conflicts later.

Reply:
It's hard to tell just from the code how these changes will alter the collar's behavior.  So my comments here are mostly about style.  @Medea-Destiny has been poking at startup times and hopefully has some more insight for you.

Reply:
I have labelled this for targeting 8.3 as the changes here are significant and will take some solid poking at and testing. In particular moving settings edit to oc_settings while very logical is going to require some solid testing -- we try to keep oc_settings as small as possible because it's where we're storing an arbitrary amount of settings data. 

@Yosty7B3 This work is really appreciated and much needed stuff, many thanks for taking a look at it. It would be helpful to talk it over some time either in world or on Discord, let me know if you'd be okay with that.

Reply:
How can I re-target this? Or should I do new pull request to target against q3?